### PR TITLE
Private tally scripts

### PR DESCRIPTION
**Pull request by danielSanchezQ**

### CODE REVIEW

File: scripts/tally/private_tallying.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    @click.option("--output", type=str, help="Output path for the merged shares file", default="./merged_proposals.shares")
+    def merge_shares(shares, output):
+        merge_generated_shares(*shares, output_file=output)
+
+    @click.command()
+    @click.option("--shares", type=str, help="Path to the merged shares file")
+    @click.option("--output", type=str, help="Output path for the merged shares file", default="./results.json")
+    def tally(shares, output):
+        tally_with_shares(shares, output)
+
```
Comment (eugene-babichenko) [inline scripts/tally/private_tallying.py:187]: Why so much empty lines? :)

Comment (danielSanchezQ) [reply scripts/tally/private_tallying.py:187]: Actually pep standard is 2 lines of separation.

Comment (eugene-babichenko) [reply scripts/tally/private_tallying.py:187]: I am well aware of PEP8. I meant the lines in the end of the file. I was commenting before you last commit.

File: scripts/tally/private_tallying.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
```
Comment (eugene-babichenko) [inline scripts/tally/private_tallying.py:2]: IIRC no need for that in Python 3, it should assume UTF-8 by default.

**Comment by danielSanchezQ** [comment]
This was implemented as part of jcli already.

**Pull request closed without merging**